src/main/java/org/snomed/snowstorm/core/util/SPathUtil.java

```diff
@@ -36,6 +40,6 @@ private static boolean isCodeSystemBranch(String branchPath) {
     }
 
     private static boolean isExpectedFormat(String branchPath) {
-        return branchPath.startsWith("MAIN") || branchPath.startsWith("SNOMEDCT-", branchPath.lastIndexOf("/") + 1);
+        return branchPath.startsWith("MAIN");
```

Comment:
No need to create method just for this.

Reply:
Thanks @CoderMChu, I have made this change.